src/main.js

```diff
@@ -156,21 +200,21 @@ function updateIdeaArray(id) {
   }
 }
 
-
 function retrieveAllStorage() {
-  checkInputFields()
+  checkInputFields();
+  checkInputFields();
   cardIDs = Object.keys(localStorage);
   for (i = 0; i < cardIDs.length; i++) {
     var parsed = JSON.parse(localStorage.getItem(cardIDs[i]));
-    var newIdea = new Idea(parsed.title, parsed.body, parsed.id, parsed.star);
+    var newIdea = new Idea(parsed);
     savedIdeas.unshift(newIdea);
   }
   renderIdeaCard(savedIdeas);
 }
 
 function filterIdeas() {
   event.preventDefault();
```

Comment:
This `event.preventDefault()` actually isn't doing what you are expecting it to do. It's a bit difficult to explain over Github as it has to do with default behaviors of HTML elements and would be very long. Let's have a meeting where I can explain a little more about why it isn't. For now, you can safely remove this line of code without it affecting functionality.